modules/installing-ocp-agent-boot.adoc

```diff
@@ -8,6 +8,22 @@
 
 Use this procedure to boot the agent image on your machines.
 
+.Prerequisites
+
+* The minimum `nmstate` version needed is:
+** `1.4.2-4` on RHEL versions greater than `8.8`
+** `2.2.7` on RHEL versions greater than `9.2`
++
+[NOTE]
+====
+On older RHEL versions update `nmstate` to version `1.4.2-4` using packages from CentOS 8 Stream by running the following command:
+
+[source,terminal]
+----
+$ sudo dnf install http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/nmstate-1.4.2-4.el8.x86_64.rpm http://mirror.centos.org/centos/8-stream/AppStream/x86_64/os/Packages/python3-libnmstate-1.4.2-4.el8.noarch.rpm
```

Comment:
This should definitely not be documented. I took this step just to confirm that a newer nmstate version worked.